.simplecov

```diff
@@ -0,0 +1,7 @@
+SimpleCov.start do
+  if RUBY_VERSION >= '2.5.0'
+    enable_coverage :branch
+    SimpleCov.root(File.join(File.dirname(__FILE__), '/lib'))
+    track_files "**/*.rb"
+  end
```

Comment:
```suggestion
  if RUBY_VERSION >= '2.5.0'
    enable_coverage :branch
  end
  SimpleCov.root(File.join(File.dirname(__FILE__), '/lib'))
  track_files "**/*.rb"
```

I think the only the `enable_coverage :branch` line may need to be limited to Ruby 2.5.0+. Perhaps the other two lines could be moved outside the conditional?

Reply:
Update: If we're only running SimpleCov on Ruby versions 2.7.0+, I think we can take out the conditional completely.